tests/test_resources.py

```diff
@@ -109,25 +115,21 @@ def test_resource_context_manager_exit(self, client):
             with SecretTestExit(name="test-context-manager-exit", namespace="default", client=client):
                 pass
 
-    def test_proxy_enabled_but_no_proxy_set(self, monkeypatch):
-        monkeypatch.setenv(name="OPENSHIFT_PYTHON_WRAPPER_CLIENT_USE_PROXY", value="1")
 
-        with pytest.raises(
-            ValueError,
-            match="Proxy configuration is enabled but neither HTTPS_PROXY nor HTTP_PROXY environment variables are set.",
-        ):
-            get_client()
+class TestClient:
+    def test_client_with_proxy(self, monkeypatch, k3scontainer_config):
+        http_proxy = "http://env-http-proxy.com"
+        monkeypatch.setenv(name="HTTPS_PROXY", value=http_proxy)
 
-    def test_proxy_conflict_raises_value_error(self, monkeypatch):
-        monkeypatch.setenv(name="OPENSHIFT_PYTHON_WRAPPER_CLIENT_USE_PROXY", value="1")
-        monkeypatch.setenv(name="HTTPS_PROXY", value="http://env-proxy.com")
+        client = get_client(config_dict=k3scontainer_config)
+        assert client.configuration.proxy == http_proxy
 
-        client_configuration = kubernetes.client.Configuration()
-        client_configuration.proxy = "http://not-env-proxy.com"
+    def test_proxy_precedence(self, monkeypatch, k3scontainer_config):
+        https_proxy = "https://env-https-proxy.com"
+        http_proxy = "http://env-http-proxy.com"
+        monkeypatch.setenv(name="HTTPS_PROXY", value=https_proxy)
+        monkeypatch.setenv(name="HTTP_PROXY", value=http_proxy)
 
-        with pytest.raises(
-            ValueError,
-            match="Conflicting proxy settings: client_configuration.proxy=http://not-env-proxy.com, "
-            "but the environment variable 'HTTPS_PROXY/HTTP_PROXY' defines proxy as http://env-proxy.com.",
-        ):
-            get_client(client_configuration=client_configuration)
+        client = get_client(config_dict=k3scontainer_config)
+        # Verify HTTPS_PROXY takes precedence over HTTP_PROXY
+        assert client.configuration.proxy == https_proxy
```

Comment:
how about test without a proxy (so if something changes we know we do not break this flow)?

Reply:
all other tests are without proxy